frontend/src/Components/Login/index.js

```diff
@@ -0,0 +1,192 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { loginUser } from '../../Redux-JS/actions/authentication';
+import classnames from 'classnames';
+import "./style.css";
+import firebase from "firebase"
+import StyledFirebaseAuth from "react-firebaseui/StyledFirebaseAuth"
+require('dotenv').config();
+
+
+// class Login extends Component {
```

Comment:
Do we need this whole block? Is the bottom the actual component or this commented portion?